sample_app/simulate_notary.py

```diff
@@ -53,8 +53,10 @@ def simulate_notary():
                                  sample_app.configure.APP_KEY)
 
     # Create identity without key. System automatic generating 3 key pairs
-    create_identity_chain_response = factom_client.identities.create(
-        ["NotarySimulation", datetime.datetime.utcnow().isoformat()])
+    create_identity_chain_response = factom_client.identities.create([
+        "NotarySimulation",
+        datetime.datetime.now(datetime.timezone.utc).isoformat().replace("+00:00", "Z")
```

Comment:
Should this also be using the `Utils.to_military_timezone_str()` function as well so that we only have the replace workaround in a single location?

Reply:
@sambarnes I didn't use the `to_military_timezone_str()` function here because I thought I would keep the "sample app" independent from the internal utilities of the SDK.

Reply:
Gotcha, thats what I figured but wanted to make sure!